samples/Basic.Tests/TestHost.cs

```diff
@@ -22,8 +22,8 @@ private IServiceProvider CreateProvider()
 		{
 			var services = new ServiceCollection();
 			services.AddLogging();
-			services
-				.AddIdentity<AppUser, AppRole>()
+			services.AddIdentityCore<AppUser>(_ => { })
+				.AddRoles<AppRole>()
 				.AddUserStore<InMemoryUserStore<AppUser, AppRole>>()
```

Comment:
Agreed that the default sample should still be AddIdentity, as it still seems to be the main use case. I could add an extra sample for Core use, but since the code is pretty much the same I don't think it's worth it.

AddIdentityCore will probably be used for apps that only provide REST API's, and don't need the web page part of the MVC package. That's where I'm using it for at least.